esmvalcore/esgf/_download.py

```diff
@@ -234,8 +235,41 @@ def same_file(result):
 
         return files
 
+    def _get_facets(self, results):
```

Comment:
Please add a docstring here

Reply:
Done in [9874227](https://github.com/ESMValGroup/ESMValCore/pull/1920/commits/9874227016311e992c5340b940547b0f1ab208b8)